src/app/models/blocks/index.js

```diff
@@ -2,42 +2,59 @@ export const rawVideoType = 'rawVideo';
 export const imageType = 'image';
 export const videoType = 'video';
 export const rawImageType = 'rawImage';
+export const listItemType = 'listItem';
+export const orderedListType = 'orderedList';
+export const unorderedListType = 'unorderedList';
 
-export const singleTextBlock = (text, id = null) => ({
-  id,
-  type: 'text',
-  model: {
-    blocks: [
-      {
-        id,
-        type: 'paragraph',
-        model: {
-          text,
-          blocks: [
-            {
-              id,
-              type: 'fragment',
-              model: {
-                text,
-                attributes: [],
-              },
-            },
-          ],
-        },
+const optionalIdBlock = (block, id) => (id ? { ...block, id } : { ...block });
```

Comment:
Just wanted to make sure I'm understanding this correctly - these `id`s are only ever relevent during tests?

If that's true, I'm a bit apprehensive about adding test functionality to our non-test code

It seems this behaviour could / should be done in the test files - as a helper like what you have here, or just directly adding IDs to the test blocks:

Before:
```js
const captionBlock = blockContainingText(
  'caption',
  'Some caption text...',
  'mocked-id'
);
```

After:
```js
const captionBlock = {
  ...blockContainingText('caption', 'Some caption text...'),
  id: 'mocked-id'
}
```

This PR is still good though - big improvement on what we had before, just curious on thoughts on this test-code-within-non-test-files specifically 👍

Reply:
I think the models may or may not include ids, and so there's the need to simulate that functionality. Right now, you're right in saying that only our tests use the id option, but I think the functionality should exist for modelling when we have ids.

Essentially, even though only tests use the optional `id` key, the `id` is an essential part of the model itself, and that option should be available in my opinion